src/v2/styles/Message/Message-layout.scss

```diff
@@ -192,13 +192,29 @@
   .str-chat__message-metadata {
     grid-area: metadata;
     display: flex;
+    flex-wrap: wrap;
```

Comment:
Adding wrap to the message metadata so that the "edited at ..." label can occupy the second line.